ui/ops/src/routes/ops/reports/Reports.vue

```diff
@@ -0,0 +1,95 @@
+<script setup>
```

Comment:
The order of all other components is:
```
<template>
<script>
<style>
```

You can edit IDEA to generate files this way via

<img width="950" height="620" alt="image" src="https://github.com/user-attachments/assets/9c92d10e-3fb3-47fd-ab5a-fd995be47f74" />